tasks/go.py

```diff
@@ -438,6 +440,57 @@ def is_excluded(pkg):
     return licenses
 
 
+@task
+def generate_protobuf(ctx):
+    """
+    Generates protobuf defintions in pkg/proto
+    """
+    base = os.path.dirname(os.path.abspath(__file__))
+    repo_root = os.path.abspath(os.path.join(base, ".."))
+    proto_root = os.path.join(repo_root, "pkg", "proto")
+
+    print("nuking old definitions at: {}".format(proto_root))
+    file_list = glob.glob(os.path.join(proto_root, "pbgo", "*.go"))
+    for file_path in file_list:
+        try:
+            os.remove(file_path)
+        except OSError:
+            print("Error while deleting file : ", file_path)
+
+    with ctx.cd(repo_root):
+        # protobuf defs
+        print("generating protobuf code from: {}".format(proto_root))
+
+        files = []
+        for path in Path(os.path.join(proto_root, "datadog")).rglob('*.proto'):
+            files.append(path.as_posix())
+
+        ctx.run(
+            "protoc -I{include_path} --go_out=plugins=grpc:{out_path} {targets}".format(
+                include_path=proto_root, out_path=repo_root, targets=' '.join(files),
+            )
+        )
+        # grpc-gateway logic
+        ctx.run(
+            "protoc -I{include_path} --grpc-gateway_out=logtostderr=true:{out_path} {targets}".format(
+                include_path=proto_root, out_path=repo_root, targets=' '.join(files),
+            )
+        )
+        # mockgen
+        mockgen_in = os.path.join(proto_root, "pbgo")
+        mockgen_out = os.path.join(proto_root, "pbgo", "mocks")
+        try:
+            os.mkdir(mockgen_out)
+        except FileExistsError:
+            pass
```

Comment:
nit for easier debugging
```suggestion
            print("{} folder already exists".format(mockgen_out))
```